### ISSUE TITLE
Lots of the new Swamp of Sorrows quests have no rewards.

### ISSUE THREAD

**Issue by nobodyinteresting** [slowtorta/turtlewow-bug-tracker#1075]
**Please, provide a clear description what the bug is:**
Obviously not every quest should have a reward, but a large chunk of the new SoS quests have none. Feels bad to go all the way from Sorrowguard to the coast, kill murlocs, then return for no reward and no continuation.

**IDs of affected creatures, items, quests or spells with a link to the relevant page:**
Sorrowguard Keep quests that don't go anywhere and have no reward:
https://database.turtle-wow.org/?quest=40072
https://database.turtle-wow.org/?quest=40079
https://database.turtle-wow.org/?quest=40076
https://database.turtle-wow.org/?quest=40078

The Harborage (Draenei) quests:
https://database.turtle-wow.org/?quest=40033
(this whole line deadends once you hit azshara, and none of the quests in it give any reward other than exp)
https://database.turtle-wow.org/?quest=40031

**Expected behavior. Describe how it should work:**
I would suggest *at least* silver rewards for these quests, if not actual items. The Sorrowguard Keep ones especially feel unrewarding, I think I did every quest there and got a total of 10 silver in rewards. Just didn't feel particularly vanilla.

**Comment by Zmitrok1984** [comment]
Each quest, listed here give XP and reputation with SW
https://database.turtle-wow.org/?quest=40072  RewXP: 2750  150 Reputation with Stormwind
https://database.turtle-wow.org/?quest=40079  RewXP: 2750  150 Reputation with Stormwind
https://database.turtle-wow.org/?quest=40076  RewXP: 2650  100 Reputation with Stormwind
https://database.turtle-wow.org/?quest=40078  RewXP: 2750  150 Reputation with Stormwind
https://database.turtle-wow.org/?quest=40033  RewXP: 5000   
https://database.turtle-wow.org/?quest=40031  RewXP: 2350

**Comment by nobodyinteresting** [comment]
Correct, but aside from exp, they give no reward at all. There are some vanilla quests that do that, usually ones that are part of chains, but very, very rarely ones that are standalone/end chains. It feels incredibly strange for an NPC to give one quest - go across the zone, kill 18 murlocs, then return - then give no reward other than exp and rep.

Picking a totally random low level quest, In Defense of the Kings Lands part 3 of 4, we find a quest that's [*part of a chain* and still manages to give *some* money/item reward.](https://classic.wowhead.com/quest=263/in-defense-of-the-kings-lands)

Again, just feels really, really weird to hand in a quest and see no money/item rewards at all, and it happens tons of times with the new Swamp of Sorrows quests.

Sorry I wasn't clearer with my title, I've edited it now.

**Comment by LeopoldHock** [comment]
This is intentional due to the final quest of the chain giving very good rewards:
https://database.turtle-wow.org/?quest=40233
